src/test/mir-opt/const_allocation/32bit/rustc.main.ConstProp.after.mir

```diff
@@ -0,0 +1,68 @@
+// MIR for `main`
+// source = MirSource { instance: Item(DefId(0:4 ~ const_allocation[317d]::main[0])), promoted: None }
+// pass_name = ConstProp
+// disambiguator = after
+
+fn  main() -> () {
+    let mut _0: ();                      // return place in scope 0 at $DIR/const_allocation.rs:7:11: 7:11
+    let _1: &[(std::option::Option<i32>, &[&str])]; // in scope 0 at $DIR/const_allocation.rs:8:5: 8:8
+    let mut _2: &&[(std::option::Option<i32>, &[&str])]; // in scope 0 at $DIR/const_allocation.rs:8:5: 8:8
+
+    bb0: {
+        StorageLive(_1);                 // bb0[0]: scope 0 at $DIR/const_allocation.rs:8:5: 8:8
+        StorageLive(_2);                 // bb0[1]: scope 0 at $DIR/const_allocation.rs:8:5: 8:8
+        _2 = const {alloc0+0: &&[(std::option::Option<i32>, &[&str])]}; // bb0[2]: scope 0 at $DIR/const_allocation.rs:8:5: 8:8
```

Comment:
Frankly we could fix this sort of output ugliness by just printing each basic block (or the entire body) first and then add the comments using the minimal indentation needed.

Feel free to open an issue about it.

Reply:
https://github.com/rust-lang/rust/issues/70327